cypress/cypress/integration/video.spec.ts

```diff
@@ -0,0 +1,145 @@
+    // write msgs
+    cy.get("[data-cy=input-field]").type("Question 1");
+    cy.get("[data-cy=input-send]").trigger("mouseover").click();
+    // display card over a corner of the video
+    cy.get("[data-cy=mentor-name-card]").should("exist");
+  });
+
+  it("Display mentor-name card & star over the left-corner of the video", () => {
+    mockDefaultSetup(cy, {
+      config: { mentorsDefault: ["clint", "carlos"] },
+      mentorData: [clint, carlos],
+      apiResponse: "response_with_feedback.json",
+      gqlQueries: [
+        cyMockGQL("UserQuestionSetFeedback", { userQuestionSetFeedback: null }),
+      ],
+    });
+    cy.intercept("**/questions/?mentor=clint&query=*", {
+      fixture: "response_with_feedback.json",
+    });
+    cy.visit("/");
+
+    // Mentor Card in the beggining
+    cy.get("[data-cy=video-container").within(($hc) => {
+      cy.get("[data-cy=mentor-name-card]").should("exist");
+      cy.get("[data-cy=mentorname-faveicon-wrapper]").within(($hc) => {
+        cy.get("[data-cy=mentor-name]")
+          .should("exist")
+          .contains("Clinton Anderson");
+        cy.get("[data-cy=fave-button]").should("exist");
+      });
+    });
+
+    cy.get("[data-cy=header]").should("have.attr", "data-mentor", "clint");
+    cy.get("[data-cy=topic-2] button").trigger("mouseover").click();
+    cy.get("[data-cy=history-chat]").should("exist");
+
+    cy.get("[data-cy=input-field]").type("user msg 1");
+    cy.get("[data-cy=input-send]").trigger("mouseover").click();
+
+    cy.get("[data-cy=history-chat").within(($hc) => {
+      cy.get("[data-cy=chat-msg-1]").contains("user msg 1");
+    });
+
+    // Mentor Card at the end
+    cy.get("[data-cy=video-container").within(($hc) => {
+      cy.get("[data-cy=mentor-name-card]").should("exist");
+      cy.get("[data-cy=mentorname-faveicon-wrapper]").within(($hc) => {
+        cy.get("[data-cy=mentor-name]")
+          .should("exist")
+          .contains("Clinton Anderson");
+        cy.get("[data-cy=fave-button]").should("exist");
+      });
+    });
+  });
+
+  it("Title Header for one mentor", () => {
+    mockDefaultSetup(cy, {
+      config: {
+        cmi5Enabled: false,
+        mentorsDefault: ["clint"],
+        styleHeaderColor: "#990000",
+        styleHeaderTextColor: "#FFFFFF",
+        styleHeaderLogo: FAKE_STYLE_HEADER_LOGO,
+      },
+      mentorData: [clint],
+    });
+    cy.viewport(1200, 700);
+    cy.intercept(FAKE_STYLE_HEADER_LOGO, { fixture: "uscheader2.png" });
+    cy.intercept("**/questions/?mentor=clint&query=*", {
+      fixture: "response_with_feedback.json",
+    });
+    cy.visit("/");
+
+    cy.get("[data-cy=header]")
+      .within(($h) => {
+        cy.get("img")
+          .should("have.attr", "src")
+          .and("eq", FAKE_STYLE_HEADER_LOGO);
+        cy.get("p").should("have.css", "color", "rgb(255, 255, 255)");
+      })
+      .and("have.css", "background-color", "rgb(153, 0, 0)")
```

Comment:
this `and` should be `should` (i'm not actually sure)?

Reply:
According to https://docs.cypress.io/api/commands/and#Syntax , .and() is an alias of .should() and works very similar